docs/roman/datamodels/datamodels_asdf.rst

```diff
@@ -0,0 +1,121 @@
+.. _datamodels_asdf:
+
+
+Working with Roman datamodels and ASDF files
+============================================
+
+If you've installed the roman calibration pipeline you should also have access to  the standalone tool asdfinfo which allows access to asdf (and roman) files
+from the terminal prompt,::
+
+    asdftool info r0000101001001001001_01101_0001_WFI16_cal.asdf
+    root (AsdfObject)
+    ├─asdf_library (Software)
+    │ ├─author (str): The ASDF Developers
+    ...
+
+also useful is::
+
+    asdftool help
+    usage: asdftool [-h] [--verbose] {help,explode,implode,extract,defragment,diff,edit,remove-hdu,info,extensions,tags,to_yaml} ...
+
+    Commandline utilities for managing ASDF files.
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      --verbose, -v         Increase verbosity
+
+    subcommands:
+
+      {help,explode,implode,extract,defragment,diff,edit,remove-hdu,info,extensions,tags,to_yaml}
+        help                Display usage information
+        explode             Explode a ASDF file.
+        implode             Implode a ASDF file.
+        extract             Extract ASDF extensions in ASDF-in-FITS files into pure ASDF files
+        defragment          Defragment an ASDF file..
+        diff                Report differences between two ASDF files
+        remove-hdu          Remove ASDF extension from ASDF-in-FITS file
+        info                Print a rendering of an ASDF tree.
+        extensions          Show information about installed extensions
+        tags                List currently available tags
+        to_yaml             Convert as ASDF file to pure YAML.
+
+
+which gives a list of possible actions one of the more useful can be::
+
+    asdftool edit file.asdf
+
+Which will open the file in an editor you have set via the EDITOR environment variable.
+
+To access the files via a python session,
+
+.. code-block:: python
+
+    import roman_datamodels as rdm
+    import numpy as np
+    import asdf
+    rdm_a = rdm.open('r0000101001001001001_01101_0001_WFI16_cal.asdf')
+    asdf_a = asdf.open('r0000101001001001001_01101_0001_WFI16_cal.asdf', mode='rw')
```

Comment:
I agree. Something like
```
When opening a file it's best to use a context manager, for example
    with rdm.open(filename) as model:
         <do stuff>
This will automatically close the file when exiting the context manager. Alternatively one can call the ``close()`` method on the datamodel object.
```

Reply:
The text has been updated.